coderd/database/dbauthz/dbauthz.go

```diff
@@ -1997,6 +1997,14 @@ func (q *querier) InsertWorkspaceAgentStat(ctx context.Context, arg database.Ins
 	return q.db.InsertWorkspaceAgentStat(ctx, arg)
 }
 
+func (q *querier) InsertWorkspaceAgentStats(ctx context.Context, arg database.InsertWorkspaceAgentStatsParams) error {
+	if err := q.authorizeContext(ctx, rbac.ActionCreate, rbac.ResourceSystem); err != nil {
+		return err
+	}
```

Comment:
review: Doing this as a system-level function. This shouldn't ever need to be called by users directly.

Reply:
Oh, shouldn't we add a message for linter informing why it is system-level access?

Reply:
Fixed